### PR TITLE
Add search shortcut support

### PR DESCRIPTION
**Pull request by ma12-co**
fix #3257
- [x] depends on: https://github.com/nextcloud/nextcloud-vue/pull/989

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -380,6 +384,10 @@ export default {
 				this.$store.dispatch('hideSidebar')
 			}
 		},
+		// Upon pressing ctrl+f, focus the search box in the left sidebar
+		handleAppSearch() {
+			document.querySelector('.conversations-search')[0].focus()
```
Comment (nickvergessen) [inline src/App.vue:389]: And now add soemthing that when it is already focused default ctrl+f happens?

Comment (ma12-co) [reply src/App.vue:389]: It already works, the second ctrl+f triggers the browser search

**Comment by skjnldsv** [comment]
> /backport to stable18

Not so sure for this anymore though. Let's make sure there is no regressions at all :)

**Comment by nickvergessen** [comment]
Setting "to develop" as dependency is still open

**Comment by ma12-co** [comment]
> Setting "to develop" as dependency is still open

Can be merged anyway imo

**Comment by nickvergessen** [comment]
But we need to bump the vue library after it was merged?

**Comment by skjnldsv** [comment]
> But we need to bump the vue library after it was merged?

the only thing missing here would be the auto-open of the sidebar, the shortcut and focus will still works

**Comment by ma12-co** [comment]
yep

File: src/App.vue [hunk 2]
```diff
@@ -20,7 +20,11 @@
 -->
 
 <template>
-	<Content :class="{ 'icon-loading': loading, 'in-call': isInCall }" app-name="Talk">
+	<Content
+		v-shortkey="['ctrl', 'f']"
+		:class="{ 'icon-loading': loading, 'in-call': isInCall }"
+		app-name="Talk"
```
Comment (skjnldsv) [inline src/App.vue:26]:
```suggestion
		app-name="talk"
```
Not needed btw :p

File: src/components/LeftSidebar/LeftSidebar.vue [hunk 3]
```diff
@@ -299,33 +300,4 @@ export default {
 	border-bottom: 1px solid var(--color-border-dark);
 }
 
-.navigation {
```
Comment (skjnldsv) [inline src/components/LeftSidebar/LeftSidebar.vue:302]: I strongly suggest you do a tour of your code to check where are the other locations you did something similar and nuke it :wink:

Comment (ma12-co) [reply src/components/LeftSidebar/LeftSidebar.vue:302]: lol, ok :)

**Comment by nickvergessen** [comment]
Needs a manual backport @ma12-co 
Can you do that?

**Pull request merged**